README.md

```diff
@@ -108,5 +113,5 @@ Your answer here.
 How is `git pull` related to `git fetch`?
 
 ```text
-Your answer here.
+fetching is the process of downloading from another repo.  pulling uses that process to integrate changes from one repo to another
 ```
```

Comment:
Not quite, what we were looking for : 

`git fetch` checks for any commits that have been added to a remote repository that you don't have in your local repository. `git pull` runs `git fetch`, and then merges those commits into the current branch of your local repository.